### ISSUE TITLE
IMX477 support revertion

### ISSUE THREAD

**Issue by Buanderie** [OE4T/linux-tegra-4.9#27]
Is there any reason for dropping the IMX477 support ? (revert from this commit: b15adf2f43095f8c207de58f1c4045b5337965e0 )

Thanks :)

**Comment by madisongh** [comment]
That's one of the changes that came from NVIDIA, you'd have to ask them for a definitive answer. I suspect, though, that they couldn't work out all the bugs/issues, or they found it interfering with the existing built-in support for other sensors (like the RPi V2 cameras). I've experimented with IMX477 support (using the [RidgeRun repo](https://github.com/RidgeRun/NVIDIA-Jetson-IMX477-RPIV3) for it), and while I could get it to (sort of) work, it would have taken a lot more effort to mix and match that sensor with others on the same system.

**Comment by Buanderie** [comment]
Oh, I see...
As a matter of fact, I adapted the RidgeRun/Leopard patches to my custom meta-tegra based build. I got it working on my Leopard carrier board (Jetson Nano production module). And when I switched to a "regular" nVidia Jetson devkit, I found out that the same driver wouldn't work anymore.
So I came here looking for a mainline driver that would work for most of the targets. Seems like they couldn't get it done properly either :)
Thanks for the input, though !
